src/components/Results.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import "./Results.css"
+
+const Results = (props) => {
+  console.log(props.results)
+  const originalUpload = props.results.pop()
```

Comment:
You sure you want to mutate the `prop.results` here? 

Ref:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop

Reply:
Ah yes, the last element in the results array is extraneous information and isn't needed for rendering the component

Reply:
Still smells bad; maybe just grab the last item instead?